libs/protos/protos/sp_internal.proto

```diff
@@ -69,13 +69,26 @@ message HeartbeatRequest {
 }
 
 message NotifyRequest {
-  string pipeline_id = 1;
-  string step_name = 2 [deprecated=true];
+  // Included in notification
+  // This will be used to pull the condition type (true, false, error) from the pipeline step,
+  // so that we can include metadata, abort condition, etc., in the notification
+  // The condition will contain the notification configuration also.
+  string condition_type = 1; // true, false, error
```

Comment:
Since the server is sending notifications, the logic to translate condition type can be in there. Can we make this be the shared step condition enum?

Reply:
Changed to enum